xml-parser/src/main/java/org/uniprot/core/xml/uniref/AbstractMemberConverter.java

```diff
@@ -0,0 +1,145 @@
+		}
+		if(uniObj.getOverlapRegion() !=null) {
+			  String overlap = uniObj.getOverlapRegion().getStart() + "-" + uniObj.getOverlapRegion().getEnd();
+			  xref.getProperty().add(createProperty(PROPERTY_SOURCE_OVERLAP, overlap));
+		}
+		if(uniObj.isSeed()!=null) {
+			xref.getProperty().add(createProperty(PROPERTY_IS_SEED, String.valueOf(uniObj.isSeed())));
+		}
+	}
+	
+
+	private PropertyType createProperty(String type, String value) {
+		PropertyType property = jaxbFactory.createPropertyType();
+		property.setType(type);
+		property.setValue(value);
+		return property;
+	}
+	protected void updateMemberFromXml(AbstractUniRefMemberBuilder<? extends AbstractUniRefMemberBuilder<?,T>, T> builder , MemberType xmlObj ) {
+		builder.memberIdType(UniRefMemberIdType.typeOf(xmlObj.getDbReference().getType()))
+		.memberId(xmlObj.getDbReference().getId());
+		
+	//	String organismName="";
+	//	Long taxId =null ;
+		
+		List<PropertyType> properties = xmlObj.getDbReference().getProperty();
+		for (PropertyType property : properties) {
+			if (property.getType().equals(PROPERTY_NCBI_TAXONOMY)) {
+				builder.organismTaxId( Long.parseLong(property.getValue()));
+			} else if (property.getType().equals(PROPERTY_PROTEIN_NAME)) {
+				builder.proteinName(property.getValue());
+			} else if (property.getType().equals(PROPERTY_SOURCE_ORGANISM)) {
+				builder.organismName(property.getValue());
+			} else if (property.getType().equals(PROPERTY_SOURCE_UNIPARC)) {
+				builder.uniparcId(new UniParcIdBuilder(property.getValue()).build());
+			} else if (property.getType().equals(PROPERTY_SOURCE_LENGTH)) {
+				builder.sequenceLength(Integer.parseInt(property.getValue()));
+			} else if (property.getType().equals(PROPERTY_SOURCE_OVERLAP)) {
+				String strOverlap = property.getValue();
+				String strStart = strOverlap.substring(0, strOverlap.indexOf('-'));
+				String strEnd = strOverlap.substring(strOverlap.indexOf('-') + 1, strOverlap.length());
+				builder.overlapRegion(
+				new OverlapRegionBuilder()
+					.start(Integer.parseInt(strStart))
+					.end(Integer.parseInt(strEnd))
+					.build()
+					);
+			} else if (property.getType().equals(PROPERTY_SOURCE_UNIREF100)) {
+				builder.uniref100Id(new UniRefEntryIdBuilder(property.getValue()).build());				
+			} else if (property.getType().equals(PROPERTY_SOURCE_UNIREF90)) {
+				builder.uniref90Id(new UniRefEntryIdBuilder(property.getValue()).build());		
+			} else if (property.getType().equals(PROPERTY_SOURCE_UNIREF50)) {
+				builder.uniref50Id(new UniRefEntryIdBuilder(property.getValue()).build());	
+			} else if (property.getType().equals(PROPERTY_SOURCE_UNIPROT)) {
+				builder.accession(new UniProtAccessionBuilder(property.getValue()).build());
+			} else if (property.getType().equals(PROPERTY_IS_SEED)) {
+				builder.isSeed(Boolean.parseBoolean(property.getValue()));
+			} else {
+				System.out.println("property.getType() = " + property.getType());
```

Comment:
can we log this with a logger (not System.out.println) , so this goes to a log file so we see it if something goes wrong?